devauth/devauth.go

```diff
@@ -1754,3 +1762,35 @@ func (d *DevAuth) updateCheckInTime(
 		}
 	}
 }
+
+func (d *DevAuth) syncCheckInTime(
+	ctx context.Context,
+	checkInTime *time.Time,
+	deviceId string,
+	tenantId string,
+) error {
+	attributes := []model.DeviceAttribute{
+		{
+			Name:        "check_in_time",
+			Description: nil,
```

Comment:
nit: if we skip this line no one will notice, right?

Reply:
done